sdks/java/extensions/zetasketch/build.gradle

```diff
@@ -0,0 +1,64 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * License); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import groovy.json.JsonOutput
+
+plugins { id 'org.apache.beam.module' }
+applyJavaNature()
+
+description = "Apache Beam :: SDKs :: Java :: Extensions :: ZetaSketch"
+
+def zetasketch_version = "0.1.0"
+
+dependencies {
+    compile library.java.vendored_guava_26_0_jre
+    compile project(path: ":sdks:java:core", configuration: "shadow")
+    compile "com.google.zetasketch:zetasketch:$zetasketch_version"
+    testCompile library.java.junit
+    testCompile project(":sdks:java:io:google-cloud-platform")
+    testRuntimeOnly project(":runners:direct-java")
```

Comment:
Do we need to depend on direct runner here? If the test is only running with dataflow runner, then `:runners:google-cloud-dataflow-java` would be enough.

Reply:
> Do we need to depend on direct runner here?

I think we do. Otherwise it you run the `./gradlew :sdks:java:extensions:zetasketch:test` task it fails.

> If the test is only running with dataflow runner ...

That's not true. Only the integration tests runs on dataflow runner. The NeedsRunner tests in `HllCountTest` runs on direct runner.